src/Fixer/Whitespace/MethodChainingIndentationFixer.php

```diff
@@ -0,0 +1,106 @@
+<?php
+
+/*
+ * This file is part of PHP CS Fixer.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *     Dariusz Rumiński <dariusz.ruminski@gmail.com>
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace PhpCsFixer\Fixer\Whitespace;
+
+use PhpCsFixer\AbstractFixer;
+use PhpCsFixer\Fixer\WhitespacesAwareFixerInterface;
+use PhpCsFixer\FixerDefinition\CodeSample;
+use PhpCsFixer\FixerDefinition\FixerDefinition;
+use PhpCsFixer\Tokenizer\Tokens;
+
+/**
+ * Fixer for rules defined in PSR2 ¶2.4.
+ *
+ * @author Vladimir Boliev <voff.web@gmail.com>
+ */
+final class MethodChainingIndentationFixer extends AbstractFixer implements WhitespacesAwareFixerInterface
+{
+    /**
+     * @var string
+     */
+    private $ident;
+
+    /**
+     * {@inheritdoc}
+     */
+    public function fix(\SplFileInfo $file, Tokens $tokens)
+    {
+        $this->ident = $this->whitespacesConfig->getIndent();
+        foreach ($tokens as $index => $token) {
+            $content = $token->getContent();
+            if ($content === '->' && $index > 0) {
```

Comment:
I can't use `$tokens[$index]->equals('->')`, because '->' token is an array
`class PhpCsFixer\Tokenizer\Token#137 (4) {
  private $content =>
  string(2) "->"


  private $id =>
  int(366)
  private $isArray =>
  bool(true)
  private $changed =>
  bool(false)
}
`
and there is a condition in equals function
`if ($this->isArray !== is_array($otherPrototype)) {
            return false;
        }`